fix esm import requirements of having extension.

bhouston opened the pull request:
For ESM and NodeNext style module resolution, you are supposed to have extensions in import statement.  There was one missing in this file.

bhouston committed:
fix esm import requirements of having extension.

donmccurdy commented:
Thanks @bhouston! I also pushed https://github.com/donmccurdy/keyframe-resample-wasm/commit/096edb214116c9fc312826aba68d194795917b9a, which should cause builds to fail if a similar mistake happens again.

donmccurdy commented:
Published v0.0.15.

bhouston commented:
Nice!  My template for optimal JS projects also just moved to "NodeNext" for module resolution. :)  If you are going ES Modules, you basically need that setting: https://github.com/bhouston/template-esbuild-ts-monorepo/commit/26d94a1fd64fa87d1122a92db3c0401d2f8839a8. I also added this ESLint rule that also targets this: https://github.com/bhouston/template-esbuild-ts-monorepo/commit/09d2670c743f626d6fa0a32d8d08289fe9c2e111
